WordPress/Classes/Services/PostService.m

```diff
@@ -786,6 +786,8 @@ - (void)updatePost:(AbstractPost *)post withRemotePost:(RemotePost *)remotePost
     post.content = remotePost.content;
     post.status = remotePost.status;
     post.password = remotePost.password;
+    post.hasVersionConflict = post.hasRevision;
```

Comment:
This looks off.  Having a revision shouldn't be enough for a post to have a version conflict.  I believe we discussed this at some point, and I also know this is a work in progress.

Can you clarify if this is changing in an upcoming PR?